Docs/install/index.rst

```diff
@@ -34,118 +34,175 @@
    #  @@-COPYRIGHT-END-@@
    # =============================================================================
 
+.. role:: hideitem
+   :class: hideitem
+
 .. _ug-installation:
 
-###################
+##################
 AIMET Installation
-###################
+##################
 
-Quick Install
-~~~~~~~~~~~~~
+This page contains instructions for the following installation scenarios:
+
+- :ref:`Quick Install of the latest version for PyTorch <idx-install-quick>`
+- :ref:`Installation of the latest version for all platforms from release packages <idx-install-latest>`
+- :ref:`Installation of older versions <idx-install-older>`
+
+You can also:
 
-The AIMET PyTorch GPU PyPI packages are available for environments that meet the following requirements:
+- :doc:`Install manually <install_host>`
+- :doc:`Install a Docker image, including from a pre-built image or local build <install_docker>`

```

Comment:
Can consider rewording as follows:
> Install or build within a docker container from a pre-built image

Reply:
How about:

> Install a pre-built or locally built image in a Docker container

Reply:
Let's word it like this:
`Install within a Docker container from a pre-built or locally built Docker image`